org/pr/installable-build.ts

```diff
@@ -0,0 +1,142 @@
+import { danger } from "danger"
+import { Status } from "github-webhook-event-types"
+import fetch from "node-fetch"
+import * as url from "url";
+import * as path from "path";
+
+const CIRCLECI_TOKEN: string = process.env['CIRCLECI_TOKEN']
+const PERIL_BOT_USER_ID: number = parseInt(process.env['PERIL_BOT_USER_ID'], 10)
+
+// This is a list of the CircleCI statuses to process
+const HOLD_CONTEXTS: string[] = ["ci/circleci: Installable Build/Hold"]
+const INSTALLABLE_BUILD_CONTEXTS: string[] = ["ci/circleci: Installable Build", "ci/circleci: Test Android on Device"]
```

Comment:
If a repo want to take advantage of this logic with a different job name, it just needs to be added here.

Reply:
This is really nice :)